workflow/rules/annotation/pass1.smk

```diff
@@ -9,6 +21,7 @@ rule snpeff:
         "logs/snpeff/{group}.log"
     params:
         reference="{}.{}".format(config["ref"]["build"], config["ref"]["snpeff_release"]),
-        extra="-Xmx4g"
+        data_dir = os.path.join(os.getcwd(), "results/databases/"),
```

Comment:
Does snpeff expect an absolute path? If so, use `lambda _, input: Path(input.db).parent.resolve()`.

Reply:
Eventually, you might need to add `str()` around the Path-statement. Depends a bit on how the wrapper uses it.